### ISSUE TITLE
Publish in nuget feed

### ISSUE THREAD

**Issue by nickb-carallon** [gitextensions/GitExtensions.GerritPlugin#4]
The gerrit plugin has been removed from Git Extensions v3.4, but this plugin does not yet appear in the nuget feed so there is no easy way to regain the gerrit functionality.

**Comment by mast-eu** [comment]
You're totally right. I need to allocate the time to finish the NuGet package. I count on next week to get it finally done.

**Comment by RussKie** [comment]
🎉 thank you
